setup.cfg

```
@@ -0,0 +1,39 @@
+[metadata]
+name = epcsunspecdemo
+version = attr: epcsunspecdemo.__version__
+description = EPC SunSpec demonstration tool
```

```suggestion
description = EPC SunSpec demonstration tool
long_description = file: README.rst
```

(and remove from `setup.py` below)